vaadin-grid-flow/pom.xml

```diff
@@ -16,8 +16,12 @@
     <name>Vaadin Grid Flow</name>
 
     <dependencies>
-
         <!-- webjars -->
+    	<dependency>
```

Comment:
```suggestion
        <!-- using an npm based webjar because it will contain the version number in the path for the resources -->
    	<dependency>
```